### DISCUSSION TITLE
unable to play in 1920x1080

### DISCUSSION THREAD

**Discussion by TVnemesis** [taysta/TaystJK#49] [Q&A] [answered]
Hello there, it's nemesis :-))
Id need a helping hand...after I recently updated my rig for new Ryzen 5600 and RTX3060 my game stucked on 1366x768 resolution

I've tried everything :
1.) /r-mode -1 with /r_customwidth(height) -> /write autoexec and reloading the game 
2.) /r_mode -2 and setting some Windows desktop resolutions -> /write autoexec & reload
3.) steps 1.&3. using Vulkan&Rend2 renderers 

still getting 1366x768 resolution..
which is WEIRD! :
1.) it does look crisp, doesnt really look like ''under 1080p reso'' and console renderer log says game runs on mode -1 and 1920 1080...
2.) it does make my view wider and furtherer on default /cg_fov 90 and camera range 80
3.) I'm on the same screen, just the GPU/CPU changed

Any suggestions here and there? Might be some odd ''Gaming'' setting in windows or Nvidia driver?

**Comment by TVnemesis** [comment, marked as answer]
I think I might have an answer already

EJK defaultly sets cg_Fov 90 and the new command /cg_fovaspectadjust 1

console says I am on /r_mode -1 1920 1080 so I beleive it
only the camera settings to be set as I like them 
in options I still see that I am using 1366x768 resolution, but fuck it XD

**Reply by taysta** [reply]
Sorry, I didn't check the discussions lately. I recommend you use `r_mode -2` to set the resolution to your desktop resolution. Otherwise to use a custom resolution, use the following commands:
`r_customWidth 1920`
`r_customHeight 1080`
`r_mode -1`
`r_fullscreen 1`
`vid_restart`
